README.md

```diff
@@ -7,6 +8,10 @@ To know more about Razorpay payment flow and steps involved, please read up here
 
 ## Installation
 
+This has 3 steps: add to project, installation and linking iOS SDK.
```

Comment:
Linking at so many places seems confusing to me. That is why I tried to play with the words a little. The last step is a very important one and is certainly linking in the true sense. Could you suggest some other terminology that fits here?